embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/storage/PayloadStorageServiceImpl.kt

```diff
@@ -132,14 +136,8 @@ class PayloadStorageServiceImpl(
      * deserializing the bytes.
      */
     override fun loadPayloadAsStream(metadata: StoredTelemetryMetadata): InputStream? {
-        return try {
-            metadata.asFile().inputStream().buffered().apply {
-                deliveryTracer?.onLoadPayloadAsStream(true)
-            }
-        } catch (exc: Throwable) {
-            deliveryTracer?.onLoadPayloadAsStream(false)
-            logger.trackInternalError(InternalErrorType.PAYLOAD_STORAGE_FAIL, exc)
-            null
+        return fileStorageService.loadPayloadAsStream(metadata).apply {
```

Comment:
`fileStorageService.loadPayloadAsStream(metadata).apply {`

This looks weird, wouldn't it throw when loadPayloadAsStream return null?

Reply:
no it won't ignore me